File: 2015/gara1/interrogazioni/att/interrogazioni.pas
function interroga(N, K: longint; var D: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    interroga := 42;
end;


const
    MAXN = 10000;
var
    N, K, i : longint;
    D       : array[0..MAXN-1] of longint;
    fr, fw  : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, K);
    for i:=0 to N-1 do
        read(fr, D[i]);

    writeln(fw, interroga(N, K, D));
    close(fr);
    close(fw);
end.

File: ed1/gara1/interrogazioni/att/interrogazioni.c
#include <stdio.h>
#include <assert.h>

#define MAXN 10000

int interroga(int N, int K, int D[]) {
    // Mettete qui il codice della soluzione
    return 42;
}

int D[MAXN];

int main() {
    FILE *fr, *fw;
    int N, K, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");

    assert(2 == fscanf(fr, "%d%d", &N, &K));
    for (i = 0; i < N; i++)
        assert(1 == fscanf(fr, "%d", &D[i]));

    fprintf(fw, "%d\n", interroga(N, K, D));
    fclose(fr);
    fclose(fw);
    return 0;
}
